src/sample/downloadController.java

```diff
@@ -32,43 +39,75 @@
   private Text pdfMessage;
 
   static String stuID;
+  static String stuName;
 
+  //Get StudentID
   public static void inputID(String text){
          stuID=text;
   }
+
+  //Get student name
+  public static void inputName(String getname){
+    stuName= getname;
+  }
+
   @FXML
-  public void handleButtonAction(ActionEvent event) throws IOException {
+  public void handleButtonAction(ActionEvent event) throws FileNotFoundException,IOException {
     final DirectoryChooser dirchooser = new DirectoryChooser();
 
     Stage stage = (Stage) anchorid.getScene().getWindow();
     File file = dirchooser.showDialog(stage);
 
-    if (file != null) {
-      List list1 = new List();
-      // input the text file for the course list
+     if (file != null) {
+
+      System.out.println("Path: " + file.getAbsolutePath());
+      textfield.setText(file.getAbsolutePath());
+
+      //Image
+      ImageData data= ImageDataFactory.create("C:/Users/user/student-course-registration-system/src/sample/images/usm-ringlogo.png");
```

Comment:
I think the file name might be incorrect

Reply:
u mean the file path?